artifactory/commands/cocoapods/podcommand.go

```diff
@@ -0,0 +1,177 @@
+package cocoapods
+
```

Comment:
This command should be moved to be inside the [`jfrog-cli-security`](https://github.com/jfrog/jfrog-cli-security) go module.

Reply:
Why? This isn't the "audit" command, I didn't see any `SetArtifactoryAsResolutionServer` functions being defined in `jfrog-cli-security`, only here (`jfrog-cli-core`)

Reply:
Will move it to reduce cross-team dependencies